src/Servers/Kestrel/test/InMemory.FunctionalTests/RequestTests.cs

```diff
@@ -57,6 +58,37 @@ public async Task StreamsAreNotPersistedAcrossRequests()
             }
         }
 
+        [Fact]
+        public async Task CallingCompleteDoesNotLogGreaterThanInfo()
+        {
+            var testContext = new TestServiceContext(LoggerFactory);
+
+            await using (var server = new TestServer(async context =>
+            {
+                context.Request.BodyReader.Complete();
+                await Task.CompletedTask;
+            }, testContext))
+            {
+                using (var connection = server.CreateConnection())
+                {
+                    await connection.Send(
+                        "POST / HTTP/1.0",
+                        "Content-Length: 11",
```

Comment:
Just send a GET so any logs about not consuming the request body don't interfere with this test.

Reply:
I do need to send a post. Why is that log not consistent? Shouldn't it either always or never be logged? 

If it isn't consistent, I can change the max loglevel to warning.

Reply:
I'm talking specifically about the RequestBodyNotEntirelyRead info-level log. I'm just pointing out that could be made higher severity later. That said, it now occurs to me that this test might be verifying the RequestBodyNotEntirelyRead log itself isn't higher than info level. Is that the case?

Reply:
The goal of this test is to make sure that an exception wasn't thrown from ConsumeAsync if the pipe is completed. I'd be fine with this test breaking if we increase the severity later.

Reply:
I don't think I read this correctly the first time. I think this is OK.